Migration step on README.md (#1)
UPRMeals/UPRMeals: README.md

## Backend

To run the back-end, run the command `yarn start:dev` from the `UPRMeals/backend` directory.

> Make sure you have the database container running and have the latest migrations applied, to do this run `yarn prisma migrate dev` inside the backend directory.

Reviewer: Can you specify how to run the database container?

```suggestion
To run the database container, ... .
Always make sure you have latest migrations applied, to apply migrations run `yarn prisma migrate dev` inside the backend directory.
```

Author: thats already on the readme in the Datase section that you created, lo muevo aqui?
